File: baml_language/crates/baml_tests/benches/cross_lang/workloads/python/mixed_ops.py
from dataclasses import dataclass


@dataclass
class Point:
    x: int
    y: int


def main() -> int:
    sum_ = 0
    s = ""
    arr: list[int] = []
    for i in range(5000):
        sum_ += i * 3 - 1
        s = s + "x"
        arr.append(i)
        p = Point(x=i, y=i + 1)
        sum_ += p.x + p.y
        if i > 2500:
            sum_ += 1
    return sum_ + len(s) + len(arr)

File: baml_language/crates/baml_tests/benches/cross_lang/workloads/go/mixed_ops.go
package crosslang

type pointMixed struct {
	x int
	y int
}

func MixedOps() int {
	sum := 0
	s := ""
	arr := []int{}
	for i := 0; i < 5000; i++ {
		sum += i*3 - 1
		s = s + "x"
		arr = append(arr, i)
		p := pointMixed{x: i, y: i + 1}
		sum += p.x + p.y
		if i > 2500 {
			sum += 1
		}
	}
	return sum + len(s) + len(arr)
}
